### DISCUSSION TITLE
Store files via FTP

### DISCUSSION THREAD

**Discussion by andremichelle** [keystonejs/keystone#9209] [Questions]
Is it possible to use FTP to store files on my server?

ChatGPT and Google Gimini are suggesting some kind of adapter, but their generated code has defective imports and it is not clear where which code goes. Any help is appreciated.

**Comment by dcousens** [comment]
No - not in any official packages anyway - out of interest why use FTP compared to say a HTTP GET/PUT API?

**Reply by andremichelle** [reply]
I do not see that a HTTP GET/PUT API is supported in keystone and I do not know why a HTTP GET/PUT API would be superior to FTP. 

Anyway, I was trying to run keystone as a "data input" framework locally, ran the db on my server and upload the files to my server. Since my server does not allow remote mysql connection already, I will try a different approach.

Thanks for your time.
